_config/graphql.yml

```diff
@@ -1,18 +1,47 @@
 ---
-Name: cmsgraphql
+Name: cms-graphql
+Only:
+  classexists: 'SilverStripe\GraphQL\Schema\Schema'
 ---
-SilverStripe\GraphQL\Manager:
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\GraphQL\Schema\Registry\ResolverRegistry:
+    constructor:
+      cmsResolver: '%$SilverStripe\CMS\GraphQL\Resolver'
+  SilverStripe\GraphQL\Schema\Registry\PluginRegistry:
+    constructor:
+      linkable: '%$SilverStripe\CMS\GraphQL\LinkablePlugin'
+SilverStripe\CMS\Model\SiteTree:
+  graphql_blacklisted_fields:
```

Comment:
Why did that change from a whitelist (`fields: [ID, LastEdited, AbsoluteLink]`) to a blacklist?

Reply:
Because this affects _all schemas_. It's an attribute of the SiteTree class itself. We need to throttle back what happens when you do `fields: '*'`, which is going to be very common with SiteTree.

Reply:
Right, I've verified that there's exceptions when you try to include blacklisted fields, which is great since it's quite an opaque setting (outside of the GraphQL schema config). I've documented the status quo via https://github.com/silverstripe/silverstripe-framework/pull/9768, but also suggested a refactor via  https://github.com/silverstripe/silverstripe-graphql/issues/322. Doesn't block this merge though.